.vscode/settings.json

```diff
@@ -43,5 +43,14 @@
     "**/lib": true,
     "**/dist": true
   },
-  "formatFiles.excludePattern": "**/*.min.js"
+  "formatFiles.excludePattern": "**/*.min.js",
+  "prettier.eslintIntegration": true,
```

Comment:
This setting seems outdated and deprecated: https://github.com/prettier/prettier-vscode/issues/999

And it is not recognized in my VSCode even:
![image](https://user-images.githubusercontent.com/6065071/97592610-6c720c80-1a33-11eb-8c7a-6feda3298d07.png)

Reply:
I am not sure about this either. I don't even know if I modified this. I didn't realize that my local vscode settings would be included in a commit. I am not sure how to fix this. I am willing to accept any pull that includes modifications to vscode settings.

Reply:
Just revert all unnecessary changes and make another commit.

Reply:
In regard to `prettier.eslintIntegration`, I share the official prettier's point of view:

- https://prettier.io/docs/en/comparison.html - “use Prettier for formatting and linters for catching bugs!”
- https://prettier.io/docs/en/integrating-with-linters.html, the first section only.
- https://prettier.io/docs/en/install.html - Install `prettier` locally into the frontend. Use `eslint-config-prettier` to make Prettier and ESLint play nice together.